includes/agentic-commerce/class-wc-stripe-agentic-commerce-integration.php

```
@@ -270,38 +270,109 @@ public function activate(): void {
 			as_schedule_recurring_action(
-				time(),
-				self::SYNC_INTERVAL,
+				$start_time,
+				$sync_interval,
 				self::SCHEDULED_ACTION,
 				[],
 				'wc-stripe'
 			);
 
-			WC_Stripe_Logger::info( 'Agentic Commerce: Scheduled recurring feed sync every ' . ( self::SYNC_INTERVAL / MINUTE_IN_SECONDS ) . ' minutes' );
+			WC_Stripe_Logger::info( 'Agentic Commerce: Scheduled recurring feed sync every ' . ( $sync_interval / MINUTE_IN_SECONDS ) . ' minutes' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Logger' in method 'schedule_recurring_feed_sync'. (undefined)

(StaticAccess)